spark-bigquery-pushdown/pushdown_common_src/main/scala/com/google/cloud/spark/bigquery/pushdowns/BigQueryStrategy.scala

```diff
@@ -51,9 +51,6 @@ class BigQueryStrategy(expressionConverter: SparkExpressionConverter, expression
     try {
       generateSparkPlanFromLogicalPlan(plan)
     } catch {
-      case ue: BigQueryPushdownUnsupportedException =>
```

Comment:
We do not want to throw an exception here because even though pushdown is unsupported for such operations (such as `CreateViewCommand`), they will be completed in Spark. An exception terminates the entire process and we don't want that to happen.

Reply:
This should be in a comment. Also, if why don't we want to fall spark it we cannot generate a spark plan due to an exception ? Create view should not be pushed down

Reply:
Added a comment in the code regarding this.

Since this is an all or nothing pushdown, all the commands are tried using the `BigQueryStrategy` first. If BigQueryStrategy returns Nil, it falls back to other Spark strategies. 

We can probably add some optimizations here to fast fail if the passed in plan contains unsupported nodes

Reply:
I have added a method `hasUnsupportedNodes()` that checks for unsupported nodes in the plan. We will need to compare performance with and without the check because with the check we are essentially going through the plan twice.